exampleCourse/questions/demo/workspace/vscode/workspace/initial_code.py

```diff
@@ -0,0 +1,2 @@
+def fib(n):
```

Comment:
I suggest renaming this file (not the function) to `fibonacci.py` or something. If it's called `starter_code.py` then it's tempting to rename it to something else.